drivers/telescope/lx200classic.cpp

```diff
@@ -322,6 +365,203 @@ bool LX200Classic::saveConfigItems(FILE *fp)
 
     IUSaveConfigNumber(fp, &MaxSlewRateNP);
     IUSaveConfigNumber(fp, &ElevationLimitNP);
+    
+    IUSaveConfigSwitch(fp, &UnparkAlignmentSP);
+
+    return true;
+}
+
+//Parking
+bool LX200Classic::Park()
+{
+    double parkAz  = GetAxis1Park();
+    double parkAlt = GetAxis2Park();
+    
+    char AzStr[16], AltStr[16];
+    fs_sexa(AzStr, parkAz, 2, 3600);
+    fs_sexa(AltStr, parkAlt, 2, 3600);
+    LOGF_DEBUG("Parking to Az (%s) Alt (%s)...", AzStr, AltStr);
+
+    double parkRA  = 0.0;
+    double parkDEC = 0.0;
+    azAltToRaDecNow(parkAz, parkAlt, parkRA, parkDEC);
+    LOGF_DEBUG("Parking to RA (%f) DEC (%f)...", parkRA, parkDEC);
+    
+    //save the current AlignmentMode to UnparkAlignment
+    LX200Generic::getAlignment();
+    int curAlignment = IUFindOnSwitchIndex(&AlignmentSP);
+    IUResetSwitch(&UnparkAlignmentSP);
+    UnparkAlignmentS[curAlignment].s = ISS_ON;
+    UnparkAlignmentSP.s = IPS_OK;
+    IDSetSwitch(&UnparkAlignmentSP, nullptr);
+    saveConfig(true, UnparkAlignmentSP.name);
+
+    if (!Goto(parkRA, parkDEC))
+    {
+        ParkSP.s = IPS_ALERT;
+        IDSetSwitch(&ParkSP, "Parking Failed.");
+        return false;
+    }
+
+    EqNP.s     = IPS_BUSY;
+    TrackState = SCOPE_PARKING;
+    LOG_INFO("Parking is in progress...");
 
     return true;
 }
+
+bool LX200Classic::UnPark()
+{
+    if (isSimulation() == false)
+    {
+        // Parked in Land alignment. Restore previous mode.
+        if (setAlignmentMode(PortFD, IUFindOnSwitchIndex(&UnparkAlignmentSP)) < 0)
```

Comment:
So my mount doesn't implement AlignmentMode *facepalm*, I will work on turning "RAMotorOff()" and "RAMotorOn()" into methods, and maybe we can integrate in the parent class.

Reply:
As my mount can be both poalr and altAz, a generic function would control both motors. So I propose to use a more generic name than RA....

Reply:
TrackingStop()/TrackingStart()?

Reply:
Sounds good